### ISSUE TITLE
Miniflow - Implementing Backpropagation with Gradient Descent - Quiz not correctly checking answer

### ISSUE THREAD

**Issue by deanofthewebb** [udacity/sdc-issue-reports#71]
The "Submit Answer" button seems to be triggering Type 2 Errors. 

A student noticed that an array of all zeros gets a false-positive.

**Comment by lachezarbozhkov** [comment]
Yes, I found the same problem. Passed the quiz with 3 different wrong kinds of answers for the gradient. 
13. Implementing Backpropagation with Gradient Descent

**Comment by gilcarmel** [comment]
I also observed the false-positive - it always passes no matter what you do.
